mindsdb/interfaces/native/learn_process.py

```diff
@@ -46,13 +47,25 @@ def run(self):
 
         to_predict = to_predict if isinstance(to_predict, list) else [to_predict]
         data_source = getattr(mindsdb_native, from_data['class'])(*from_data['args'], **from_data['kwargs'])
+
+        try:
+            original_proc_title = setproctitle.getproctitle()
+            setproctitle.setproctitle('mindsdb_native_process')
+        except Exception:
+            pass
+
         try:
             mdb.learn(
                 from_data=data_source,
                 to_predict=to_predict,
                 **kwargs
             )
-        except Exception as e:
+        except Exception:
+            pass
+
+        try:
+            setproctitle.setproctitle(original_proc_title)
+        except Exception:
```

Comment:
This should be set when the learn process starts and unset when it ends (or rather, no need to ever unset it)

Reply:
That is because we don't want to e.g. start learning, then the tilte changes, and then the predictor never gets put in the database or dumped to s3